src/main/java/com/wishlist/controllers/ProductController.java

```diff
@@ -0,0 +1,57 @@
+package com.wishlist.controllers;
+
+import com.wishlist.exceptions.ProductNotFoundException;
+import com.wishlist.model.Product;
+import com.wishlist.services.ProductService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/product")
```

Comment:
In REST controllers we want to expose different resources in a standardized way. Read more about it:
https://www.restapitutorial.com/lessons/httpmethods.html
https://www.restapitutorial.com/lessons/restfulresourcenaming.html
https://docs.microsoft.com/en-us/azure/architecture/best-practices/api-design
https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/